kafka-inmemory/resources/kubernetes.yaml

```diff
@@ -1,3 +1,91 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: barnabas-cluster-controller
+  labels:
+    app: barnabas
+---
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1beta1
```

Comment:
just a nit ... can be follow the same sequence apiVersion and kind instead of the opposite as in this case ?